### DISCUSSION TITLE
Is it possible to apply cross validation on complete dataset instead on training set in pycaret?

### DISCUSSION THREAD

**Discussion by dryogibansal** [pycaret/pycaret#2072] [Q&A]
Hi,

I have separate train and test sets. My dataset is very small and don't want to further divide it into train/test with cross validation in the setup function. instead, I wanted to use my whole training set to evaluate on test set. Cross validation in training set only seems a skeptical concept instead on whole dataset as in cases where train set is very small, we are further reducing the train set with cross validation. So, is it possible way to avoid cross validation in training set or do cross validation on whole dataset or just stick to using complete training set to evaluate on test set? Kindly enlighten me here.

**Comment by ngupta23** [comment]
Maybe you can set the test size to a very small value of 0.01 (1% of data)?
